spec/unit/mixin/homebrew_user_spec.rb

```diff
@@ -47,6 +47,8 @@ class ExampleHomebrewUser
     let(:user) { nil }
     let(:brew_owner) { 2001 }
     let(:default_brew_path) { "/usr/local/bin/brew" }
+    let(:default_brew_path_arm) { "/opt/homebrew/bin/brew" }
+    let(:default_brew_path_linux) { "/home/linuxbrew/.linuxbrew/bin/brew" }
```

Comment:
we never use either of these...

Reply:
added 2 more unit test blocks for those paths.